### ISSUE TITLE
Update adminDataset form with custom field components

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1558]
## User story

These forms lack a11y features, e.g. required inputs lack required attribute
- http://gigadb.gigasciencejournal.com:9170/adminDataset/create
- http://gigadb.gigasciencejournal.com:9170/adminDataset/update/id/5

It should be relatively straightforward to replace the existing inputs with custom input components that have built-in a11y features (TextField, etc)
